### ISSUE TITLE
Investigate improvements to Command Line access

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#713]
I've been reading a page of command-line guidance: https://clig.dev/

There is more guidance here: https://medium.com/@jdxcode/12-factor-cli-apps-dd3c227a0e46

I have these take-aways, so far:
* we should be able to run app via `pepys_admin` and `pepys_import` on Macos & Win.  These will require different config steps.  It's quite easy to include this in windows `install_pepys.bat`.
* if no parameters are provided, provide help
* _maybe_ switch to single `pepys` command, followed by what to do: `pepys import`, `pepys admin`, `pepys viewer`.  in this way, users will could just start with `pepys` and it will provide the next commands that can be used.
* we _could_ use a conversational interface for adding new elements:
```
> pepys add platform
name: NELSON
identifier: F334
platform-type: <drop-down list>
Adding this platform:
Name: Nelson
Identiifer: F335
...
1. Add 
2. Make more changes
3. Cancel
```
* Consider ASCII infographics for summary status. (somehow...)
![image](https://user-images.githubusercontent.com/1108513/101825733-f2e73700-3b25-11eb-850f-160b39430464.png)

[to be continued]

**Comment by arunlal-v** [comment]
With reference to the point on "Conversational interface ...", I strongly believe the tool should offer both interfaces, i.e. the conversational one and the silent powermode utility that does its job and returns without any chatter giving only the confirmation after the job is done.
And this can be achieved in the same script with minimum to no additional effort. If the options are passed as parameters, the tool will know to operate in powermode, else it'll prompt for required missing input.

**Comment by IanMayo** [comment]
Yes, @arunlal-v - good suggestion.  Providing command line options for import values would allow power users to create a batch file of multiple imports with parameters that could have been derived (for example) from the data filenames.
